Library/test-helpers/lib.sh

```diff
@@ -1253,6 +1253,78 @@ limeCreateTestDir() {
 }
 
 
+true <<'=cut'
+=pod
+
+=head2 limeCopySourceFile
+
+Copy file to actual dir from specified path or download file 
+from repository to actual dir.
+
+    limeCopySourceFile
+
+=over
+
+=back
+
+Returns 0 when the when copy of file to actual dir is succesfull.
+
+=cut
+
+limeCopySourceFile(){
+
+    local OPTION=$1
+    local FILEPATH=$2
+
+    if [ ${OPTION} == "--c" ]; then
+        cp ${FILEPATH} . && chmod a+x ${NAME}
+    elif [ ${OPTION} == "--i" ]; then
+        curl -O https://raw.githubusercontent.com/keylime/keylime/master/scripts/${NAME} && chmod a+x ${NAME}
+    fi
+
+}
+
+true <<'=cut'
+=pod
+
+=head2 limeGetSourceFile
+
+Trying to find the keylime script file in several directories on the system, the possible path is stored in
+the variables and then are tested to see where the file is located. When the condition is met 
+limeCopySourceFile is called, which copies the file from the default location to the current directory. 
+If no file is found on the system, limeCopySourceFile is called and the file is downloaded from the repository
+to the current directory.
+
+    limeGetSourceFile
+
+=over
+
+=back
+
+Returns 0 when the when getting copy of file to actual dir is succesfull.
+
+=cut
+
+limeGetSourceFile(){
+
+    NAME=$1
+    local SOURCELOCAL=$(find /usr/local/lib -name ${NAME})
+    local SOURCELIB=$(find /usr/lib/python* -name ${NAME})
```

Comment:
I would use just a single `find` command such as
```
find /usr/lib/python* /usr/local/lib/python* -print 2> /dev/null | grep pluggy/hooks.py | head -1
```
This would be for the `-i` (default) parameter.
For the `-s` parameter the `find` would search in `/var/tmp/keylime_sources`.